nsqadmin/http.go

```diff
@@ -354,6 +361,13 @@ func (s *httpServer) nodeHandler(w http.ResponseWriter, req *http.Request, ps ht
 		messages = append(messages, pe.Error())
 	}
 
+	for _, producer := range producers {
+		// Assume it is an ipv6 address if a colon is in it
+		if strings.Contains(producer.BroadcastAddress, ":") {
+			producer.BroadcastAddress = fmt.Sprintf("[%s]", producer.BroadcastAddress)
```

Comment:
you should not apply the work-around here, this makes `producers.Search(node)` just below not find anything, and result in the 404 "NODE_NOT_FOUND"